plugins/node/src/tasks/node.ts

```diff
@@ -38,13 +38,20 @@ export default class Node extends Task<typeof NodeSchema> {
       throw error
     }
 
+    const { execArgv } = process
+    // disable native fetch if supported by runtime
+    if (process.allowedNodeEnvironmentFlags.has('--no-experimental-fetch')) {
```

Comment:
**praise**: nice, i didn't know this existed, thought we'd have to resort to version sniffing